tests/acceptance/not-found-test.js

```diff
@@ -0,0 +1,32 @@
+import { visit } from '@ember/test-helpers';
+import percySnapshot from '@percy/ember';
+import { a11yAudit } from 'ember-a11y-testing/test-support';
+import { setupMirage } from 'ember-cli-mirage/test-support';
+import { setupApplicationTest } from 'ember-qunit';
+import loadDefaultScenario from 'ember-website/mirage/scenarios/default';
+import { setupPageTitleTest } from 'ember-website/tests/helpers/page-title';
+import { module, test } from 'qunit';
+
+module('Acceptance | not found', function (hooks) {
+  setupApplicationTest(hooks);
+  setupMirage(hooks);
+  setupPageTitleTest(hooks);
+
+  hooks.beforeEach(function () {
+    loadDefaultScenario(this.server);
+  });
+
+  test('Percy snapshot', async function (assert) {
+    await visit('/foo-bar-baz');
+    await percySnapshot(assert);
+
+    assert.ok(true);
+  });
+
+  test('Accessibility audit', async function (assert) {
+    await visit('/foo-bar-baz');
+    await a11yAudit();
+
+    assert.hasPageTitle('Page Not Found - Ember.js');
+  });
+});
```

Comment:
I wanted to add a test that checks that the status code of 404 is returned. I didn't figure out to, however, and didn't find information online or on Discord. Would you like to give it a try?

If it takes too much time to figure out, I think we can assume that FastBoot will correctly set the status code.

Reply:
Added `ember-cli-fastboot-testing` for this. Hope that's okay. It had to be different test though.

Reply:
Yep, sounds good.